src/api/TestExchange.ts

```diff
@@ -0,0 +1,37 @@
+import BigNumber from 'bignumber.js';
+import { BaseExchangeApi, ExchangeApi } from '../types/ExchangeApi';
+
+type RateRecord = Record<string, string>;
+
+const rates: Record<string, RateRecord> = {
+    TKN: {
+        USD: '0.0775886924643371'
+    },
+    RBTC: {
+        USD: '23345.834630269488'
+    }
+};
+
+const CURRENCY_MAPPING: Record<string, string> = {
+    TKN: 'TKN',
+    RBTC: 'RBTC'
+};
+
+export class TestExchange extends BaseExchangeApi implements ExchangeApi {
```

Comment:
I guess this test class is meant to support the test token TKN. But I still don't think we should provide such a class. Instead, it should be up to whoever implements the Pricer to mock a response. Also if you are including this class, it should still be tested, but you probably didn't include a test because this is a test class by itself.

Reply:
correct I did not test it because its a test class but I could copy the coinBase test's that could test this class. Im missing unit testing for the abstract class but it would also be the same test of coinBase. 

Since we are also implementing the pricer, this TestExchange was created to mock the price of the TKN that we use as testing. Or even we could not implement it and just map the TKN with RIF.

Reply:
@ironFe93 We ended up implementing this for two main reasons:
- to support our TestToken with the RelayPricer
- to start creating the structure to support later on different exchange APIs.

I know that a possible alternative could have been just having the value hard-coded without creating the additional class, but in this way we are following the same structure will be followed when a new exchange API will be implemented.